<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileSystemDialog</class>
 <widget class="QDialog" name="FileSystemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FileSystemDialog</string>
  </property>
  <widget class="QTreeView" name="dirView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>281</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="fileView">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>201</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="selectButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
